src/Subsystems/Puncher.cpp

```diff
@@ -32,6 +36,18 @@ void Puncher::SlingShot(double mSpeed)
 
 }
 
+double Puncher::GetPuncherLimit() {
+  return pullSling1.GetSensorCollection().GetPulseWidthPosition();
```

Comment:
`pullSling1.GetSensorCollection().GetPulseWidthPosition()` is not correct. Let's work out what this really is.